README.md

```diff
@@ -0,0 +1,126 @@
+# spark-rewards
+
+This implements revision D of [Meridian: Off-chain scheduled rewards](https://spacemeridian.notion.site/Meridian-Off-chain-scheduled-rewards-f9480ef009464ecfaf02a4f752cc4d70).
+
+## Routes
+
+### `GET /scheduled-rewards`
+
+Response:
+
+```js
+{
+  "address1": "scheduledRewards1",
+  "address2": "scheduledRewards2",
+  // ...
+}
+```
+
+### `GET /scheduled-rewards/:address`
```

Comment:
We are working with addresses in `0x` format only, is that correct? It would be great to mention that at the top of the README file.

Reply:
All of our systems are only working with the `0x` format, do you think this is important to clarify here?